rc/tools/spell.kak

```diff
@@ -3,6 +3,8 @@ declare-option -hidden str spell_last_lang
 declare-option -hidden str spell_tmp_file
 
 declare-option -docstring "default language to use when none is passed to the spell-check command" str spell_lang
+declare-option -docstring "spell checking engine to use" str spell_cmd 'aspell --byte-offsets -a -l'
```

Comment:
`makecmd` and `grepcmd` and `lintcmd` and `formatcmd` and `termcmd` all lack an underscore, so `spell_cmd` probably should too.

Reply:
Sure.